tests/stub_virtual_machine.h

```diff
@@ -106,12 +107,11 @@ struct StubVirtualMachine final : public multipass::VirtualMachine
     {
     }
 
-    void add_vm_mount(const std::string&, const VMMount&) override
-    {
-    }
-
-    void delete_vm_mount(const std::string&) override
+    std::unique_ptr<MountHandler> make_native_mount_handler(const SSHKeyProvider* ssh_key_provider,
+                                                            const std::string& target, const VMMount& mount) override
     {
+        // TODO replace with StubMountHandler when writing tests
+        return nullptr;
```

Comment:
Until then, throwing might be a better option (easier to catch if this is called inadvertently).